Response-Errors.md

```diff
@@ -0,0 +1,104 @@
+| `BLOCKCHAIN_ERROR` | `2118` | Invalid signer. The transactions were not signed by the Shopper's private key | The signer of the transactions (his public address) should be equal to the shopper's `walletAddress` | Processing payment |
+| `BLOCKCHAIN_ERROR` | `2119` | Cannot decode generic transaction data! | You will get this error if a provided signed transaction is not valid or the parameter types provided on payment creation are not valid | Processing payment |
+    
+### 3. Not Found Errors
+    
+| Error Name            | Code  | Message    | Description | Оccurrence |
+| --------------------- | ----- |--------------- | -------- | -------- |
+| `NOT_FOUND_ERROR` | `3010` | Payment was not found | The requested payment does not exist | Getting payment |
+| `NOT_FOUND_ERROR` | `3011` | Payment document was not found | You will get this error if you try to preview invoice/receipt or send invoice for a payment which is not successful | Preview invoice/receipt / send invoice |
+| `NOT_FOUND_ERROR` | `3012` | Vendor was not found  | The requested vendor does not exist | Getting vendor |
+| `NOT_FOUND_ERROR` | `3013` | Shopper was not found | The requested shopper does not exist | Getting shopper |
+| `NOT_FOUND_ERROR` | `3014` | API User was not found! | The requested API user does not exist | Getting api user |
+| `NOT_FOUND_ERROR` | `3015` | User was not found |The requested user does not exist | Getting user |
+| `NOT_FOUND_ERROR` | `3018` | The record you have searched for does not exists | You will get this error if your searching criteria is invalid | Every time we can not parse searching criteria |
+
+
+### 4. Validation Errors
+
+| Error Name            | Code  | Message    | Description | Оccurrence |
+| --------------------- | ----- |--------------- | -------- | -------- |
+| `VALIDATION_ERROR` | `4011` | Invalid property **{property}**! **{property}** cannot be null, empty or whitespace | You will get this error when you do not provide a required property in the request | Everywhere where there is request body with required properties |
+| `VALIDATION_ERROR` | `4012` | Missing **{request body object}** data | You will get this error when you do not provide a required object in the request | Everywhere where there is request body with required objects |
+| `VALIDATION_ERROR` | `4013` | Property **{property}** should be an array of objects  | You will get this error when a property is array of objects, but you provide something different | Everywhere where there is request body with array of objects type properties |
+| `VALIDATION_ERROR` | `4111` | Nonexistent country code | You will get this error if you pass non-existing country code to Vendor, CardHolderInformation | Processing Fiat payment / calculating VAT  |
+| `VALIDATION_ERROR` | `4211` | (**Input property**) should be true/false | You will get this error if you pass non-boolean value to CardHolderInformation.isCompany for example | Processing payment |
```

Comment:
You will get this error if you pass non-boolean value to CardHolderInformation.isCompany for example
->
`You will get this error if you pass non-boolean value to a property which is required to be boolean`